src/rewriter/rw_parser.py

```diff
@@ -266,25 +201,50 @@ def var_list(self):
                  lambda s, l, t: self.var_decl_action(t[0], t[1], t[2:])))
         return (pp.Suppress('(') + pp.ZeroOrMore(decl) + pp.Suppress(')'))
 
+    def def_list(self):
+        d = pp.Suppress((pp.Suppress('(') + self.symbol() + self.expr()
+                         + pp.Suppress(')'))
+                .setParseAction(lambda s, l, t: self.def_decl_action(t[0], t[1])))
+        # The pp.Keyword is suppressed since otherwise it will go into the mkrewrite mechanism
```

Comment:
What does `will go into the mkrewrite mechanism` mean here?

Reply:
misleading comment. deleted